src/main/resources/portal.properties.EXAMPLE

```diff
@@ -37,6 +37,7 @@ skin.documentation.dat=Authenticating-Users-via-Tokens.md
 
 # setting controlling the logos
 skin.right_logo=
+skin.left_logo=
 skin.tag_line_image=tag_line.png
```

Comment:
I usually prefer commenting out properties as it makes it easier to control defaults in the future. Also for testing: we currently won't test what happens when the property is not set (as it uses EXAMPLE properties), which is going to be the default for anybody upgrading their instance

Reply:
@inodb Ok, I matched the `skin.right_logo` entry. Are these different, or should I also comment out that one?